joomla/joomla-cms
# View property cleanup

### Summary of Changes

This adds view properties and cleans up doc blocks for said properties across all our core components. This is the one place we have been really really bad at keeping things in check in recent years
### Testing Instructions

You need to test every single view in Joomla still works as before. The only real way this will break something is if a view property should have been public instead of protected which would give a fatal error. So it will be really obvious if something is wrong.
### Further Notes

All new PR's that add view properties MUST now create a view property with it

I currently have skipped 1 view in Joomla Update and the Media Manager component because I honestly had no clue what half the properties actually did
